phpcs.xml.dist

```diff
@@ -9,17 +9,13 @@
 	<exclude-pattern>vendor/*</exclude-pattern>
 	<exclude-pattern>components/Markdown.php</exclude-pattern>
 
+	<config name="minimum_supported_wp_version" value="4.5"/>
 	<rule ref="WordPress-Core"/>
 	<rule ref="WordPress-Docs"/>
 	<rule ref="WordPress-Extra"/>
-	<rule ref="PHPCompatibility"/>
 
-	<rule ref="WordPress.PHP.DiscouragedPHPFunctions">
-		<!-- From "Extra": The create_function group is excluded as WP core still supports PHP 5.2 and 5.2 does not support anonymous functions. -->
-		<properties>
-			<property name="exclude" value="create_function"/>
-		</properties>
-	</rule>
+	<config name="testVersion" value="5.3-"/>
```

Comment:
Why remove exclusion for DiscouragedPHPFunctions rule?

Reply:
With the project as PHP 5.3+, removing this exclusion will flag current and future instances of `create_function` that should be replaced with anonymous functions.

Reply:
We don't use it directly, but the Markdown PHP parser we use does. Will need to look at using a different markdown parser in the future.

Reply:
Since it only seems to be a [single instance](https://github.com/pods-framework/pods/blob/a404a389c192849ded643c3a0925de213c5e1fdd/components/Markdown.php#L1996), and [edits](https://github.com/pods-framework/pods/commits/a404a389c192849ded643c3a0925de213c5e1fdd/components/Markdown.php) have been made to that embedded dependency, I'd suggest adding a `// phpcs.ignore WordPress.PHP.DiscouragedPHPFunctions` to the end of the line (or an empty line before it), so that you're explicitly dealing with _just that instance_ of `create_function()`, and not other instances that might be introduced elsewhere.